### PR TITLE
[Minor fix] Fix sub-bullets and image links

### PR DESCRIPTION
**Pull request by CBroz1**
- Current spacing renders sub-bullets as one pdf block: [live site](https://datajoint.com/docs/core/concepts/getting-started/diagrams/). This PR separates the sub-bullets as shown in the following set on line types
- Current links to images are not rendering. Fixing relative paths

### CODE REVIEW

File: src/core/concepts/getting-started/diagrams.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ between them. The `draw` method plots the graph, with many other methods (
 [Matlab](https://github.com/datajoint/datajoint-matlab/blob/master/%2Bdj/ERD.m)) to
 save or adjust the output.
 
-Because DataJoint pipelines are directional (see [DAG](../../../glossary#dag)), the tables at the
+Because DataJoint pipelines are directional (see [DAG](../../../glossary#dag)), the 
+tables at the
```
Comment (guzman-raphael) [inline src/core/concepts/getting-started/diagrams.md:14]: Was this intentional? Seems a bit funky to add a newline here and make this line short.

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
@CBroz1 Thanks for updating these broken references!

This looks great and only had a minor point on some a line break that seemed unintentional.

File: src/core/concepts/getting-started/diagrams.md (hunk 1 above)
Comment (CBroz1) [reply src/core/concepts/getting-started/diagrams.md:14]: Yes, I've been intentionally adding single line breaks to comply with the [markdown linter](https://github.com/DavidAnson/markdownlint). The motivation is that hard wrapping makes it easier to see small changes in future PRs. Should we change the [linter settings](https://github.com/datajoint-company/datajoint-docs/blob/main/.markdownlint.yaml)?

**Review by dimitri-yatsenko — APPROVED** [review]

File: src/core/concepts/getting-started/diagrams.md (hunk 1 above)
Comment (CBroz1) [reply src/core/concepts/getting-started/diagrams.md]: Notes: @guzman-raphael and I discussed the shortcomings of this markdown linter. For PRs, it's useful to see separate editing and linting doesn't fit with the standard practice of a deterministic linter that runs for every commit.

@guzman-raphael - I've added another commit to fix the wrapping for this pgf. Please reapprove so we can merge here.

**Review by dimitri-yatsenko — APPROVED** [review]

**Review by guzman-raphael — APPROVED** [review]

**Pull request merged**